tests/rados/test_rados_preempt_scrub.py

```diff
@@ -18,210 +19,236 @@
-                )
-                return 1
-            mon_obj.set_config(
-                section="osd", name="osd_shallow_scrub_chunk_max", value="250"
-            )
-            mon_obj.set_config(section="osd", name="osd_scrub_chunk_max", value="250")
-            mon_obj.set_config(section="osd", name="debug_osd", value="10/10")
-
-            rados_object.bench_write(
-                pool_name=entry["pool_name"], rados_write_duration=300, background=True
-            )
-
-            log_lines = [
-                "head preempted",
-                "WaitReplicas::react(const GotReplicas&) PREEMPTED",
-            ]
-
-            init_time, _ = installer.exec_command(cmd="sudo date '+%Y-%m-%d %H:%M:%S'")
-
-            rados_object.run_scrub(pool=entry["pool_name"])
-            scrub_status_flag = False
+        init_time, _ = installer.exec_command(
+            cmd="sudo date '+%Y-%m-%dT%H:%M:%S.%3N+0000'"
+        )
+        init_time = init_time.strip()
+        rados_object.configure_pg_autoscaler(**{"default_mode": "off"})
+        method_should_succeed(rados_object.create_pool, **config)
+
+        if not rados_object.bench_write(
+            pool_name=pool_name, rados_write_duration=20, max_objs=15, num_threads=1
+        ):
+            log.error(f"Failed to write objects into {pool_name}")
+            return 1
+        acting_set = rados_object.get_pg_acting_set(pool_name=pool_name)
+        log.info(f"The PG acting set is -{acting_set}")
+        set_parameter_value(mon_obj, rados_object, pool_name)
 
-            # Wait for the scrub to start for 15 minutes
-            time_end = time.time() + 60 * 45
-            count = 1
-            while time.time() < time_end:
-                scrub_status = check_scrub_status(rados_object)
-                if scrub_status is True:
-                    scrub_status_flag = True
-                    log.info("Scrubbing is in progress")
-                    break
-                log.info("Waiting for the scrub to start")
-                time.sleep(30)
-                count = count + 1
-                log.info(f"The count is -{count}")
-            total_time = count * 30
-            log.info(
-                f"The total time to start user initiated scrub is  - {total_time} seconds"
+        object_list = rados_object.get_object_list(pool_name)
+        rados_object.run_deep_scrub(pool=pool_name)
+        time.sleep(5)
+        for _ in range(30):
+            tmp_date = subprocess.check_output("date", text=True).strip()
+            for object in object_list:
+                if re.search(r".*object.*", object):
+                    cmd_rewrite = (
+                        f"echo {tmp_date} | rados -p {pool_name} put {object} -"
+                    )
+                    rados_object.client.exec_command(cmd=cmd_rewrite, sudo=True)
+        get_paramter_value(mon_obj, rados_object, pool_name)
```

Comment:
Let's update the method name to reflect the things done. name can be `get_preempt_parameter_value`

Reply:
Updated the method name.